arcade/examples/gui_disappearing_info_box.py

```diff
@@ -0,0 +1,83 @@
+"""
+Example code showing how to use the DisappearingInfoBox
```

Comment:
```suggestion
Disappearing Info Box

An example of how to make info boxes vanish after some amount of time.
```
Having the user friendly title in the header is the convention for the doc and marginally helpful for spot checking examples from the terminal in vim.